test/widgets/action_sheet_test.dart

```diff
@@ -24,6 +24,7 @@ import '../api/fake_api.dart';
 import '../example_data.dart' as eg;
 import '../flutter_checks.dart';
 import '../model/binding.dart';
+import '../model/internal_link_test.dart';
 import '../model/test_store.dart';
```

Comment:
nit: Import internal_link_test.dart to use setupstore to generate link without an '/' before '#'.